boards/pulga/include/periph_conf.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2022 Universidade de São Paulo
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_pulga
+ * @{
+ *
+ * @file
+ * @brief       Peripheral configuration for the Pulga board
+ *
+ * @author      Geovane Fedrecheski <geonnave@gmail.com>
+ *
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "kernel_defines.h"
+#include "periph_conf_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    SPI configuration
+ * @{
+ */
+static const spi_conf_t spi_config[] = {
+    {
+        .dev  = NRF_SPIM0,
+        .sclk = GPIO_PIN(0, 8),
+        .mosi = GPIO_PIN(0, 4),
+        .miso = GPIO_PIN(0, 6),
+    }/*, XXX: apparently, the second SPI is not routed in Pulga X V1.0
+    {
+        .dev  = NRF_SPIM2,
+        .sclk = GPIO_PIN(0, 22),
+        .mosi = GPIO_PIN(0, 17),
+        .miso = GPIO_PIN(0, 20),
+    }*/
+};
+#define SPI_NUMOF           ARRAY_SIZE(spi_config)
+/** @} */
+
+/* Config for which pins are used for uart */
+#ifndef PULGA_UART0_RX
+#define PULGA_UART0_RX GPIO_PIN(0,25)
```

Comment:
Why do these get a special define?

Reply:
They are being used, by convention, as the default pins for UART.